notebooks/create_example_data.py

```diff
@@ -107,11 +107,11 @@ def multiply_database_size(filename: str, multiplier: int = 2):
             )
 
 
-# loop through 5 times, copying the database and
+# loop through 6 times, copying the database and
```

Comment:
Do you mean 8?

Reply:
Great catch, thanks, I'll remove the numeric mention here and defer to the range specification as it could change.